apps/public-docsite-v9/src/Concepts/Migration/FromV8/Troubleshooting.stories.mdx

```diff
@@ -43,6 +43,11 @@ function App() {
 }
 ```
 
+## "I tried uing a layered v8 component like Callout in a layered v9 component like Dialog and my Callout appears beneath my Dialog. What's going on?"
```

Comment:
Following our offline conversation, solution with `PortalMountNodeProvider` might be preferred.

Reply:
Agreed. Updated to recommend that rather than overriding `z-index` styles.